packages/react/src/components/Tree/Tree.tsx

```diff
@@ -146,9 +146,35 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
   }
 
   static getAutoControlledStateFromProps(nextProps: TreeProps, prevState: TreeState) {
-    const itemsForRender = Tree.getItemsForRender(nextProps.items)
+    const { items } = nextProps
+    const itemsForRender = Tree.getItemsForRender(items)
+    let { activeItemIds } = nextProps
+
+    if (!activeItemIds && items) {
+      activeItemIds = prevState.activeItemIds
+
+      const expandedItemsGenerator = items =>
```

Comment:
This currently doesn't work if the second item inside the Tree has `expanded`. May need to be refactored to something like:
```
  static expandedItemsGenerator = (items, activeItemIds) =>
  _.reduce(
    items,
    (acc, item) => {
      if (item['expanded'] && acc.indexOf(item['id']) === -1) {
        acc.push(item['id'])
      }

      if (item['items']) {
        return [...acc, Tree.expandedItemsGenerator(item['items'], acc)]
      }

      return acc
    },
    activeItemIds,
  )

  static getAutoControlledStateFromProps(nextProps: TreeProps, prevState: TreeState) {
    const { items } = nextProps
    const itemsForRender = Tree.getItemsForRender(items)
    let { activeItemIds } = nextProps

    if (!activeItemIds && items) {
      activeItemIds = prevState.activeItemIds
      activeItemIds = Tree.expandedItemsGenerator(items, activeItemIds)
    }

    return {
      activeItemIds,
      itemsForRender,
    }
  }
```